spring-webserver/server-java/src/main/java/net/corda/server/NodeRPCConnection.java

```diff
@@ -0,0 +1,51 @@
+package net.corda.server;
+
+import net.corda.client.rpc.CordaRPCClient;
+import net.corda.client.rpc.CordaRPCConnection;
+import net.corda.core.messaging.CordaRPCOps;
+import net.corda.core.utilities.NetworkHostAndPort;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+/**
+ * Wraps a node RPC proxy.
+ *
+ * The RPC proxy is configured based on the properties in `application.properties`.
+ *
+ * @param host The host of the node we are connecting to.
```

Comment:
The @params comments are marked red in IntelliJ. Java doesn't support @param at class level comment. Probably you might want to more this to individual fields.

Reply:
Moved the Javadoc tags to constructor for Java version.